pkg/model/api/v1/jsonref.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+	"regexp"
+)
+
+var jsonRefRegexp = regexp.MustCompile(`^#/([a-zA-Z0-9_-]+)(?:/([a-zA-Z0-9_-]+))*$`)
```

Comment:
well looks like I have an issue with the question `How to capture multiple repeated groups`.
Quite interesting actually, but not really how to solve it.

The problem currently is that:

for the reference `#/spec/panels/name` it will capture `spec` and `name` not the `panels`. Which looks like a common mistake called `Repeating a Capturing Group vs. Capturing a Repeated Group`

Reply:
mmm it's not solvable like that, but I could actually simplify the thing and it should solve the issue.

Reply:
voilà should be better now